src/components/popups-etc/session-conflict-warning-modal.tsx

```diff
@@ -0,0 +1,39 @@
+import Button from '../design-system/button';
+import styles from './session-conflict-warning.module.css';
+import { showSaveConflictModal, continueSaving } from '../../lib/state';
+
+interface SessionConflictWarningModalProps {
+    sessionId: string;
+	gameId: string;
+}
+
+export default function SessionConflictWarningModal({ sessionId, gameId }: SessionConflictWarningModalProps) {
+    const close = () => {
+        showSaveConflictModal.value = false;
+    };
+
+	const handleOverwrite = () => {
+		localStorage.setItem('lastSavedSessionId', JSON.stringify({ sessionId, gameId: gameId }));
+		console.log(localStorage.getItem('lastSavedSessionId'));
+		showSaveConflictModal.value = false;
+		continueSaving.value = true;
+	};	
+
+    return (
+        <div className={styles.overlay} style={{ display: showSaveConflictModal.value ? 'flex' : 'none' }}>
+            <div className={styles.modal}>
+                <h2>Session Conflict!</h2>
+                <p>
+                    It seems you have another session open with changes. Would you like to overwrite the changes from the other session or reload to retrieve those changes to this tab?
+                </p>
+                <div className={styles.buttonGroup}>
+                    <Button onClick={close}>Cancel</Button>
+                    <Button accent onClick={handleOverwrite}>Overwrite</Button>
+                    <Button accent onClick={() => {
```

Comment:
just wondering - why offer a reload button?

Reply:
Josias suggested on adding 3 button, 

cancel = disable autosave and do nothing they can edit whatever delete etc and when they do ctrl + s we ask again if they wana overwrite or reload or cancel 

reload = it's to get latest info so they don't overwrite or go to previous tab they just reload so they get the latest version of code where they can countiune

Overwrite = it overwrite whatever old session might be and autosave the current changes on tab aka sending Save function from that tab = overwriting if anything was written on new one and old one is closed 


hope this explained.